en/mkdocs.yml

```diff
@@ -79,10 +77,6 @@ nav:
         - 'Overview': quick-starts/single-sign-on.md
         - 'OAuth/OIDC Apps': quick-starts/sso-for-oidc-apps.md
         - 'SAML Apps': quick-starts/sso-for-saml-apps.md
-      - 'Multi-Factor Authentication': 
-        - 'Possession Factor':  
-          - 'SMS OTP': quick-starts/mfa-sample.md
```

Comment:
why is this removed? this doc is referred in some other places as well

Reply:
Nexmo app is outdated. So this doc needs to be tested with their rebranded version, https://dashboard.nexmo.com/getting-started/sms

Reply:
since this doc is referred in multiple places, I think it is better if this can be tested. Also, we need this doc as we have a link to this in the landing page (for try it out).

Reply:
This is referred in the corresponding guide and I have remove the dependencies now. 
Ofcourse, we need to test it but I don't think we should have it in the nav till then.

Reply:
let's keep it commented